pkg/cache/clusterqueue.go

```diff
@@ -680,21 +681,30 @@ func (c *ClusterQueue) UsedCohortQuota(fName kueue.ResourceFlavorReference, rNam
 	return cohortUsage
 }
 
-// DominantResourceShare returns a value from 0 to 100 representing the maximum of the ratios
+// DominantResourceShare returns a value from -100 to 100 representing the maximum of the ratios
 // of usage above nominal quota to the lendable resources in the cohort, among all the resources
 // provided by the ClusterQueue.
+// If negative, it means that the usage of the ClusterQueue is below the nominal quota.
 // The function also returns the resource name that yielded this value.
 func (c *ClusterQueue) DominantResourceShare(w *workload.Info) (int, corev1.ResourceName) {
```

Comment:
Two suggestions for readability:
1.  `DominantResourceShare(wl)` -> `DominantResourceShareWith(wl*)`
2. `DominantResourceShare()` to denote current `DominantResourceShare(nil)`